<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1592</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 250, 255);</string>
    </property>
    <property name="title">
     <string>DATA READ FROM STM32</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_readData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="text">
      <string>Read Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 245, 255);</string>
    </property>
    <property name="title">
     <string>MODE</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SETPOINT:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_velocity">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 150, 0);</string>
     </property>
     <property name="text">
      <string>VELOCITY (rpm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_setpoint">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_position">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 150, 0);</string>
     </property>
     <property name="text">
      <string>POSITION (degrees)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 235, 255);</string>
    </property>
    <property name="title">
     <string>SET COM</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_disconnect">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>DISCONNECT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PORT:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_COM">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Baud">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Baudrate:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Select_COM">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reload">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>RELOAD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status_com">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>DISCONNECT</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(244, 244, 255);</string>
    </property>
    <property name="title">
     <string>RUN AND RESET</string>
    </property>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>VNI-Korin</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 55, 55), stop:1 rgba(155, 155, 155,155));
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}
</string>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
     <property name="icon">
      <iconset theme="applications-games"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>VNI-Korin</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 55, 55), stop:1 rgba(155, 155, 155,155));
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}
</string>
     </property>
     <property name="text">
      <string>RUN</string>
     </property>
     <property name="icon">
      <iconset theme="applications-games"/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(229, 235, 255);
</string>
    </property>
    <property name="title">
     <string>PID PARAMETERS</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_Kd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kd:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Kp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Ki">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Kp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Kp:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Kd">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Ki">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ki:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendPID">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(140, 0, 255);</string>
     </property>
     <property name="text">
      <string>SEND PID</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/Admin/Downloads/in-komi-we-trust.jpg</normaloff>C:/Users/Admin/Downloads/in-komi-we-trust.jpg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="PID2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 167);</string>
     </property>
     <property name="text">
      <string>PID2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PID1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 167);</string>
     </property>
     <property name="text">
      <string>PID1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 250, 255);</string>
    </property>
    <property name="title">
     <string>COMMAND</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTextBrowser" name="textBrowser_command">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>911</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Limit_Time</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>891</width>
        <height>611</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
       </font>
      </property>
      <property name="title">
       <string>Graph 1</string>
      </property>
      <widget class="QCustomPlot" name="customPlot1" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>871</width>
         <height>551</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_plot">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>580</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(0, 170, 0);
</string>
       </property>
       <property name="text">
        <string>PLOT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clearGraph">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>580</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Graph</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Real_Time</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>891</width>
        <height>611</height>
       </rect>
      </property>
      <property name="title">
       <string>Graph 2</string>
      </property>
      <widget class="QCustomPlot" name="customPlot2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>871</width>
         <height>551</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_plot_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>580</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 255);</string>
       </property>
       <property name="text">
        <string>PLOT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clearGraph_2">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>580</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Graph</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>680</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(210, 210, 255);
border-color: rgb(0, 0, 255);
color: rgb(60, 131, 255);</string>
    </property>
    <property name="text">
     <string>NHÓM 5 - NGỌC HOÀN - THẾ HOAN - HỮU TOÀN - HỮU PHÚC</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1592</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPID">
    <property name="title">
     <string>PID</string>
    </property>
   </widget>
   <addaction name="menuPID"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_txt"/>
  </widget>
  <action name="actionSave_txt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/save_file.png</normaloff>:/img/save_file.png</iconset>
   </property>
   <property name="text">
    <string>SAVE TXT - GRAPH</string>
   </property>
   <property name="toolTip">
    <string>SAVE TXT - GRAPH</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
